lib/driver.js

```diff
@@ -450,16 +451,14 @@ class YouiEngineDriver extends BaseDriver {
       retryCount++;
       connected = await connectedPromise;
 
-      if (!connected && errno === 'ECONNREFUSED') {
+      if (!connected && (errno === 'ECONNREFUSED' || errno === -61)) {
```

Comment:
On node v14+ errno is now a negative number representing the error itself (in this case -61 is ECONNREFUSED)

Reply:
This good to know